terragraph_planner/common/rf/link_budget_calculator.py

```diff
@@ -502,44 +502,16 @@ def get_tx_power_from_rsl(rsl_dbm: float, net_gain_dbi: float) -> float:
     return rsl_dbm - net_gain_dbi
 
 
-def get_link_capacity_from_snr(
-    snr_dbm: float, mcs_snr_mbps_map: List[MCSMap]
-) -> Tuple[int, float]:
+def get_link_capacity_from_mcs(
+    mcs_level: int, mcs_snr_mbps_map: List[MCSMap]
+) -> float:
     """
-    Returns MCS and Mbps values given the signal-to-noise
-    (OR signal to interference + noise) ratio in dBm.
-
-    @param snr_dbm: the SNR or SINR value in dBm
-    @param mcs_snr_mbps_map: table of SNR thresholds and Mbps
-
-    returns MCS and the Mbps values
+    Returns Mbps values given the MCS level
     """
-    mcs_level = get_mcs_from_snr(snr_dbm, mcs_snr_mbps_map)
     mbps_value = [row.mbps for row in mcs_snr_mbps_map if row.mcs == mcs_level]
```

Comment:
nit: we can change `List[MCSMap]` to `Dict[int, float]` for quick lookup, but it may change a lot of code? It can be a separate BE task.

Reply:
Probably Dict[int, some type] where some type has snr/mbps/tx_backoff. Too much trouble to do here, so maybe in the future.